tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot

```diff
@@ -63,6 +63,39 @@ Upgrade VCH with containers
     Should Not Contain  ${output}  Rolling back upgrade
     Should Be Equal As Integers  ${rc}  0
 
+    # check version
```

Comment:
Yeah, because what we do is: upgrade, check version, roll back, check version, upgrade, check version. It's intentional. It'd be clearer if I broke the steps out into a callable but Robot is such a pain LOL